checks/alertmanager.go

```diff
@@ -88,3 +89,25 @@ func extractMessage(annotations map[string]string) string {
 	}
 	return ""
 }
+
+func generateNameSuffix(name string, labels map[string]string) string {
+	// Based on the type of alert, we will add alert metadata to the check name
+	// The key here is the alert type and the name will be "/".join(value)
+	nameSuffixFromLabels := map[string][]string{
+		"KubePodCrashLooping":            {"namespace", "pod"},
+		"KubePodNotReady":                {"namespace", "pod"},
+		"KubeDeploymentReplicasMismatch": {"namespace", "deployment"},
+		"KubeJobFailed":                  {"namespace", "job_name"},
+		"KubeletTooManyPods":             {"node"},
+		"KubeContainerWaiting":           {"namespace", "pod", "container"},
+	}
+
+	if val, exists := nameSuffixFromLabels[name]; exists {
+		var suffix []string
+		for _, v := range val {
+			suffix = append(suffix, labels[v])
+		}
+		return name + "-" + strings.Join(suffix, "/")
```

Comment:
```suggestion
		return name + "/" + strings.Join(suffix, "/")
```

Reply:
@moshloop This would lead to the name being `KubePodCrashLooping/prod/backend-admin-app-7b58d4f6b7-h4629` instead of `KubePodCrashLooping-prod/backend-admin-app-7b58d4f6b7-h4629`

Do we want that ? There is no distinction b/w alert type and id then

Reply:
I think it reads better with a `/` rather than 2 different seperators